JustSaying.Tools/JustSaying.Tools.csproj

```diff
@@ -1,6 +1,6 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
@martincostello Locally I get a nuget warning about `Magnum` being restored using full framework, which turns into an error due to `TreatWarningsAsErrors`. I see that CI is passing and am a little confused.

Reply:
Also, this project is using the async methods in the AWS SDK, switching TFMs will require some small code changes to build as far as I can see. This project is probably superseeded internally at JustEat with JustSayingWeb (or whatever these days).

Reply:
Blargh - I'll look at it later.

Reply:
I'm not familiar with `Magnum` and it seems to be only used for commandline parsing in `JustSaying.Tools` and hasn't been updated in ages (much like `JustSaying.Tools` ).

Suggest that we stop using it, in favour of more  modern / better maintained approaches to same?

Reply:
Agreed. I'm actually working on that now with the new command line package from Microsoft (experimental, but why not?). The only thing we lose is the interactive nature that's there currently (i.e. type "quit" to exit etc...), but I think it's a bit weird and not useful.

Reply:
I think this would be a straight forwards port to the "DotnetMetal" model, see the sample here:
https://github.com/dotnet/command-line-api/blob/b20e71a404eaac6cde023706336723fff60f28ef/samples/DotnetMetal/CommandDefinitions.cs

Reply:
Yep, [that's the successor that I meant](https://github.com/dotnet/command-line-api).

Reply:
For now I've just reverted this in 74e0fef167b118a5c7f73b4604d412d7bcebebe2 - we can update the TFM and fix all the associated issues another time.